src/bento/userLoginData.js

```diff
@@ -0,0 +1,26 @@
+export const pageTitle = 'Login';
+
+export const loginProvidersData = {
+  google: {
+    key: 'google',
+    icon: 'https://raw.githubusercontent.com/CBIIT/datacommons-assets/main/bento/images/icons/png/google.png',
+    loginButtonText: 'Sign in with Google',
```

Comment:
Can we remove "Sign in with" or "Sign in" from each button?

Reply:
Buttons should now only say the name of the IDP being used for logging in